tests/namespaced.rs

```diff
@@ -266,6 +266,42 @@ mod namespaced {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn sandwich_waypoint_plain() -> anyhow::Result<()> {
+        let mut manager = setup_netns_test!(InPod);
+
+        let _zt = manager.deploy_ztunnel(DEFAULT_NODE).await?;
+
+        let waypoint = manager
+            .workload_builder("waypoint", DEFAULT_NODE)
+            .mutate_workload(|w| {
+                w.application_tunnel = Some(ApplicationTunnel {
```

Comment:
This doesn't really do anything, is it just here for explicitness?

Reply:
TBH thought it did matter... I was thinking of adding a PROXY test as well but didn't get around to it